rosette/api.py

```diff
@@ -460,19 +460,24 @@ def call(self, parameters):
         @return: A python dictionary expressing the result of the invocation.
         """
 
-        if not isinstance(parameters, _DocumentParamSetBase):
```

Comment:
Removing this allows "all" (list, tuple, set, etc...) variable types past, which previously would have failed once the request is sent to RS with a badrequestformat. Now they fail with "'list' object has no attribute 'use_multipart'". If we already using input type validation we should probably raise an incompatible error message for non string and DocumentParamaters to provide a more user friendly error message